test/test_datasets_download.py

```diff
@@ -223,103 +184,84 @@ def root():
 
 
 def places365():
-    return itertools.chain(
-        *[
-            collect_download_configs(
-                lambda: datasets.Places365(ROOT, split=split, small=small, download=True),
```

Comment:
We ever only used the callable to parametrize the dataset class call. Thus, we can simplify this by accepting the class as well as the args and kwargs. This also allows us to automatically detect the dataset module, since this is available through the class. Note that we can't use `functools.partial` here for this reason.